spec/support/matchers/csv_column_matcher.rb

```diff
@@ -0,0 +1,56 @@
+# Examples:
+# expect(report).to have_column("Quantity")
+# expect(report).to have_column("Quantity").with_value("1")
+# expect(report).to have_column("Quantity").with_values("1", "2")
+# expect(report).to have_column("Quantity").with_values(["1", "2"])
+RSpec::Matchers.define :have_column do |column_header|
+  match do |report|
+    lines = report.to_csv.split("\n")
+    headers = lines.shift.split(",")
+    @column_index = headers.index(column_header)
+    break false unless @column_index
+    break true unless @expected_values
+    @actual_values = lines.map { |line| line.split(",")[@column_index].to_s }
```

Comment:
Thanks, that's definitely nicer. 

Looks like we need `headers: true` as an option in the `CSV.parse` call.